help_to_heat/frontdoor/schemas.py

```diff
@@ -702,7 +705,9 @@ class SessionSchema(Schema):
     postcode = fields.String(
         validate=validate.Regexp(postcode_regex_collection, error=_("Please enter a valid UK postcode"))
     )
-    uprn = fields.Integer()
+    uprn = fields.String()
+    rrn = fields.String()
+    epc_details = fields.Dict()
```

Comment:
I think I still have some fields to add here, but changes will probably be ongoing until the work is complete. If my understanding is correct and this is just supposed to be a record of all the properties in the session data.